packages/stryker/src/stryker-cli.ts

```diff
@@ -14,6 +15,7 @@ program
   .usage('<command> [options] [stryker.conf.js]')
   .description(`Possible commands: 
     run: Run mutation testing
+    init: Initalize Stryker for your project
```

Comment:
The latest and greatest stryker-cli should work like this for _every_ command:

* Is stryker installed locally?
   * No: prompt to install locally
* Pipe command to installed stryker

It shouldn't care about the actual command being executed, lets leave that up to stryker.